[CloudPC]add singleSignOn into ProvisioningPolicy and OrganizationSetting (#19104)
microsoftgraph/microsoft-graph-docs: api-reference/beta/resources/cloudpcorganizationsettings.md

|Property|Type|Description|
|:---|:---|:---|
|enableMEMAutoEnroll|Boolean|Specifies whether new Cloud PCs will be automatically enrolled in Microsoft Endpoint Manager(MEM). The default value is `false`.|

> |enableSingleSignOn|Boolean|True if the provisioned Cloud PC can be accessed by single sign-on option, false if the provisioned Cloud PC don't support this feature. Default value is false. Through single sign-on, Windows 365 users can authenticate to Azure AD with passwordless options (like FIDO keys) to access their Cloud PC.|No|No|No|

Reviewer: Please update all description to: 
```suggestion
|enableSingleSignOn|Boolean|Describle......When TRUE, indicate...When FALSE, indicate... Default value is.......Through single sign-on, Windows 365 users can authenticate to Azure AD with passwordless options (like FIDO keys) to access their Cloud PC.|No|No|No|
```
